src/renderer/components/dock/install-chart/chart-data/call-for-helm-chart-values.injectable.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { getChartValues } from "../../../../../common/k8s-api/endpoints/helm-charts.api";
+
+export type CallForHelmChartValues = (
+  repo: string,
+  name: string,
+  version: string
+) => Promise<string>;
+
+const callForHelmChartValuesInjectable = getInjectable({
+  id: "call-for-helm-chart-values",
+  instantiate: (): CallForHelmChartValues => getChartValues,
```

Comment:
Shouldn't this also be `causesSideEffects`?

Reply:
Sure, good catch.

Reply:
Fixed